packages/dev/core/src/Shaders/geometry.fragment.fx

```diff
@@ -97,7 +97,7 @@ void main() {
             gl_FragData[DEPTH_INDEX] = vec4(vViewPos.z / vViewPos.w, 0.0, 0.0, 1.0);
         #endif
 
-        #ifdef PREPASS_NORMAL
+        #if defined(PREPASS_NORMAL) || defined(PREPASS_WORLD_NORMAL)

```

Comment:
The wgsl geometry.fragment should also be updated

Reply:
Done.